src/components/NumberedCounter/NumberedCounter.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {ActionIcon, Group, Text} from '@mantine/core';
+
+const NumberedCounter = ({value, limit, onIncrement, onDecrement}: NumberedCounterType) => {
+
+    const incrementHandler = () => {
+        if (value < limit) {
+            onIncrement(value + 1)
+        }
+    }
+    const decrementHandler = () => {
+        if (value > 1) {
+            onDecrement(value - 1)
+        }
+    }
+
+    return (
+        <Group spacing={5}>
```

Comment:
a lot of `magic numbers` spacing 5 what it means ? i have no idea=) it can be grouped somehow -? I **strongly** recommend use classes and not use inline styles